app/views/eventos/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="container">
+  <h3>Cadastrar novo evento</h3>
+  <%= simple_form_for @evento do |f| %>
+    <%= f.input :nome, required: true, label: "Nome" %>
+    <%= f.input :data_inicio, required: true, label: "Data início", as: :date, html5: true %>
+    <%= f.input :data_fim, required: true, label: "Data fim", as: :date, html5: true %>
+    <%= f.input :local, required: true, label: "Local" %>
+    <%= f.input :endereco, required: true, label: "Endereço" %>
+    <%= f.input :site, required: true, label: "Site" %>
+    <%= f.input :intro, label: "Introdução" %>
+    <%= f.input :modalidades, required: true, label: "Modalidades" %>
+    <%= f.input :cronograma, label: "Cronograma" %>
+    <%= f.input :resultados, label: "Resultados" %>
+    <%= f.input :inscricoes, required: true, label: "Inscrições" %>
+    <%= f.button :submit, "Salvar", class: "btn btn-success" %>
+
```

Comment:
Mesma coisa do edit. Na verdade acho que era melhor a gente fazer um partial _form e reaproveitar. Eu não fiz na notícia pq o /new tem só o título, e também o formulário é muito simples. Mas nesse caso do evento seria legal.